providers/shared/components/apigateway/id.ftl

```diff
@@ -75,6 +75,33 @@ object.
                 "Type" : STRING_TYPE,
                 "Default" : ""
             },
+            {
+                "Names" : "Logging",
+                "Children" : [
+                    {
+                        "Names" : "Execution",
+                        "Children" : [
+                            {
+                                "Names" : "Enabled",
+                                "Description" : "Logging of the API Gateway Execution Logs.",
+                                "Type" : BOOLEAN_TYPE,
+                                "Default" : false
+                            }
+                        ]
+                    },
```

Comment:
I don't think we should include this one. The execution log is a troubleshooting log and should be mandatory for all deployments. The access logging is optional as that can have significant cost implications

Reply:
If this is removed you could also make it an Object shorter and make it 

AccessLogging with children of Enabled

Reply:
Thanks, updated in b973911 and 8bff92b